apis/core/hydra/dataset.ttl

```diff
@@ -16,7 +16,7 @@ void:Dataset
         hydra:Operation,
         schema:UpdateAction ;
       hydra:title
-        "Update dataset" ;
+        "Edit cube metadata" ;
```

Comment:
This is not exactly cube metadata either 🤔 🤷 
```suggestion
        "Edit metadata" ;
```

Reply:
I just changed that!
We always talked about "cube metadata" with customers, so that's why I think it makes sense to use the same wording here.